appengine/flexible/django_cloudsql/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = mysite.settings
+python_files = tests.py
```

Comment:
I think you wanted to have this file because 1) passing an env var and 2) let pytest pick up `tests.py`.

For 1, you can copy noxfile_config.py from the git root
```
cp ../../../noxfile_config.py .
```
Then add the envvar to `envs` as follows:
```python
    'envs': {'DJANGO_SETTINGS_MODULE': 'mysite.settings'},
```

For 2, I suggest you rename the test file to `test_polls.py` or something.

That way, you don't have to have your own `pytest.ini`.

If you really want to have `pytest.ini`, let me know. You need to include the contents in `pytest.ini` at the git root too.

Reply:
Done.